src/define.js

```diff
@@ -19,7 +18,7 @@ export default function (Ctor) {
     // If we used defineProperty() then the consumer must also use it and
     // cannot use property initialisers. Instead we just set it so they can
     // use whatever method of overridding that they want.
-    Ctor.is = uniqueId();
+    Ctor.is = `x-${uniqueId()}`;
```

Comment:
Sort of. `prefix` within `uniqueId()` tightly coupled it to this use case. Didn't want that as it's also used in the `sym` function now.